packages/discovery-provider/src/tasks/publish_scheduled_releases.py

```diff
@@ -12,6 +19,73 @@
 batch_size = 100
 
 
+def create_remix_contest_notification(session: Session, track: Track):
```

Comment:
great add

Reply:
could be nice to share function, but all good

Reply:
i tried but the paradigm was different.. lemme look again

Reply:
oh wait no it's totally do-able